app/web/templates/admin/dashboard.html

```diff
@@ -236,10 +236,10 @@ <h3 class="card-title">{{ _('Create New Server') }}</h3>
                     </div>
 
                     <div class="form-group has-feedback">
-                      <label>
-                          {{ _('Server Path') }} - <small>{{ _('Example /var/opt/minecraft/server') }}</small>
+                      <label>      
+                          {{ _('Server Path') }} - <small>{{ _('Example: /Crafty_Controller_Fabric/server/server_1/2/3/4') }}</small>
                         </label>
-                        <input type="text" class="form-control" placeholder="{{ _('/var/opt/minecraft/server') }}" name="server_path" maxlength="255" value="/var/opt/minecraft/server">
+                        <input type="text" class="form-control" placeholder="{{ _('/Crafty_Controller_Fabric/server/server_1') }}" name="server_path" maxlength="255" value="/Crafty_Controller_Fabric/server/server_1">
```

Comment:
Likewise, watch the hardcoding. Don't shoot yourself in the foot

Reply:
these are just placeholder values, also idk how to write html lol